src/apps/vpn/purchasewebhandler.cpp

```diff
@@ -44,6 +44,8 @@ void PurchaseWebHandler::startSubscription(const QString&) {
           &MozillaVPN::abortAuthentication);
   connect(taskAuthenticate, &TaskAuthenticate::authenticationCompleted, vpn,
           &MozillaVPN::completeAuthentication);
+  connect(taskAuthenticate, &TaskAuthenticate::authenticationCompleted, this,
+          &PurchaseWebHandler::subscriptionDone);
```

Comment:
Instead of calling this method, just emit the signal. Replace `subscriptionDone` with `subscriptionCompleted`